src/LibraryManager.Vsix/Constants.cs

```diff
@@ -12,6 +12,9 @@ public static class Constants
         public const string TelemetryNamespace = "webtools/librarymanager/";
         public const string MainNuGetPackageId = "Microsoft.Web.LibraryManager.Build";
         public const string ErrorCodeLink = "https://github.com/aspnet/LibraryManager/wiki/Error-codes#{0}";
+        public const string WAP = "{349C5851-65DF-11DA-9384-00065B846F21}";
+        public const string WEBSITE_PROJECT = "{E24C65DC-7377-472B-9ABA-BC803B73C61A}";
```

Comment:
Nit: Should we make the constant name mixed case to match the rest of the constants? I.e. WebsiteProject rather than WEBSITE_PROJECT?